installer/install.sh

```diff
@@ -32,6 +32,7 @@ kops_env=""
 domain=""
 gce_zone="us-west1"
 quiet="no"
+imm_release_prefix="imm"
```

Comment:
why you create default value for this prefix, it already exist in `main.sh`. The previous approach was good